runtime/nls/j9cl/j9jcl.nls

```diff
@@ -588,3 +588,20 @@ J9NLS_JCL_CRIU_FAILED_TO_ENABLE_ALL_RESTORE_OPTIONS.explanation=NOTAG
 J9NLS_JCL_CRIU_FAILED_TO_ENABLE_ALL_RESTORE_OPTIONS.system_action=
 J9NLS_JCL_CRIU_FAILED_TO_ENABLE_ALL_RESTORE_OPTIONS.user_response=
 # END NON-TRANSLATABLE
+
+# Note: "java.compiler" is a system property name and should not be translated.
+J9NLS_JCL_JAVA_COMPILER_WARNING_XJIT=Setting the java.compiler system property is obsolete in version 21 and later, use -Xjit instead.
```

Comment:
This should note that `-Xjit` is a command line parameter not to be translated.

Reply:
The comment is unnecessary, the AI is smart enough not to translate it, but I will add it.

Reply:
Doing translations is now done by machine and not by any person.